influxdb3/batching/batcher.go

```diff
@@ -151,3 +196,16 @@ func (b *Batcher) emitPoints() []*influxdb3.Point {
 
 	return points
 }
+
+// Flush drains all points even if the internal buffer is currently larger than size.
+// It does not call the callbackEmit method
+func (b *Batcher) Flush() []*influxdb3.Point {
+	slog.Info(fmt.Sprintf("Flushing all points (%d) from buffer.", b.CurrentLoadSize()))
+	points := b.points
+	b.points = b.points[len(points):]
```

Comment:
Why such emptying? Why not `b.points = b.points[:0]`

Reply:
When starting the project, I wanted a Flush method to reset everything and start again.  The slice indexing on line 205 was the first to occur to me.  It worked and I forgot about it.  However the suggested slice is more efficient.  Updated.